source/reference/options.txt

```diff
@@ -468,6 +474,52 @@ Authentication Options
    - ``none``, the effect is the same as setting
      ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.
 
+.. option:: --oidcFlows
+
+   Specifies OpenID Connect flows in a comma-separated list. 
+   The OpenID connect flows specify how ``mongosh``interacts with the identity 
```

Comment:
```suggestion
   The OpenID connect flows specify how ``mongosh`` interacts with the identity 
```

The space should fix the markup/rendering weirdness.

Reply:
Done.